README.md

```diff
@@ -1 +1,221 @@
-# math-magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+Math Magicians
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] It would be nice if you fix the Live Demo and FAQ links because these links are broken right now 😨 Kindly fix them or remove them to make your readme descriptive and professional 👍